app/controllers/campaign_controller.rb

```diff
@@ -39,7 +39,7 @@ def update
   def show
     campaign = Campaign.find(params[:id])
     respond_to do |format|
-      format.json { render json: campaign }
+      format.json  { render :json => campaign.to_json(:methods => [:cover_photo_url, :description_photo_url, :profile_photo_url]) }
```

Comment:
Rubocop will not like this line cause of the use of the hashrocket/fat comma (=>).

Please use Ruby 2 hash syntax.